run_analysis.sh

```diff
@@ -4,6 +4,15 @@ ECOSYSTEM="$1"
 PACKAGE="$2"
 PKG_PATH="$3"
 
+NOPULL=0
```

Comment:
Perhaps it would be easier to use `shift` rather than positional parameters.

Reply:
Done (along with other stuff)